build.sbt

```diff
@@ -12,7 +12,7 @@ licenses := Seq("Apache 2.0" -> url("https://raw.githubusercontent.com/amplab/ke
 
 homepage := Some(url("http://keystone-ml.org"))
 
-scalaVersion := "2.10.4"
+scalaVersion := "2.10.6"
```

Comment:
Not much. Its just a change to build file and using `+package` / `+test` to run both versions. If test pass and you can see both artifacts (i.e. `target/scala-2.11/<name>.jar` and `target/scala-2.10/<name>.jar`) you should be set